features/0434-outofband/README.md

```diff
@@ -82,25 +80,34 @@ The *receiver* state machine is as follows:
 
 [![receiver state machine google doc](state-machine-receiver.png)](https://docs.google.com/spreadsheets/d/1o1szGCMx6FzrxfAKV76IEYmeKRJqvV3gd1r3Qfv1Q9o/edit#gid=10874521)
 
-Worth noting is the first event of the `done` state, where the receiver may receive the message multiple times. This represents, for example, an agent returning to the same website and being greeted with the same QR code each time.
+Worth noting is the first event of the `done` state, where the receiver may receive the message multiple times. This represents, for example, an agent returning to the same website and being greeted with instances of the same QR code each time.
 
 ### Messages
 
-There are two out-of-band protocols, each consisting of a single message. In both cases, the message is emitted by the *sender*.
+The out-of-band protocol a single message that is sent by the *sender*.
 
-#### Message Type: `https://didcomm.org/oob-invitation/%VER/invitation`
+#### Message Type: `https://didcomm.org/out-of-band/%VER/invitation`
 
 ```jsonc
 {
-  "@type": "https://didcomm.org/oob-invitation/%VER/invitation",
+  "@type": "https://didcomm.org/out-of-band/%VER/invitation",
   "@id": "<id used for context as pthid>",
   "label": "Faber College",
   "goal-code": "issue-vc",
   "goal": "To issue a Faber College Graduate credential",
-  "protocols": [
+  "setup_protocols": [
       "https://didcomm.org/didexchange/1.0",
       "https://didcomm.org/connections/1.0"
-      ]
+      ],
+  "request~attach": [
+    {
+        "@id": "request-0",
+        "mime-type": "application/json",
+        "data": {
+            "base64": "<bytes for base64>"
```

Comment:
if we use `json` instead of `base64` data encoding, we avoid the 33% bloat incurred with re-base64 encoding something.

Reply:
I'll update.  We had the same issue in the "issue credential" v2.0 exchange.  Thanks.

Reply:
I didn't take that example as normative. Should I? How the sender encodes the attachment seems out of scope of this RFC.